client/src/components/Profile/EditProfile.tsx

```diff
@@ -0,0 +1,184 @@
+                id="username"
+                margin="normal"
+                InputLabelProps={{
+                  shrink: true,
+                }}
+                InputProps={{
+                  classes: { input: classes.inputs },
+                }}
+                name="firstname"
+                autoComplete="firstname"
+                autoFocus
+                placeholder="Your First Name"
+              />
+              <TextField
+                className={classes.testWidth}
+                id="username"
+                margin="normal"
+                InputLabelProps={{
+                  shrink: true,
+                }}
+                InputProps={{
+                  classes: { input: classes.inputs },
+                }}
+                name="lastname"
+                autoComplete="lastname"
+                autoFocus
+                placeholder="Your Last Name"
+              />
+              <Select labelId="label" id="select" value="10" className={classes.dropDown}>
+                <MenuItem value="10">Select</MenuItem>
+                <MenuItem value="20">Male</MenuItem>
+                <MenuItem value="20">Female</MenuItem>
+              </Select>
+              <Box flexDirection="row" className={classes.phoneDetails}>
+                <Select className={classes.dropDown} id="selectMonth" value="0">
+                  <MenuItem value="0">Month</MenuItem>
+                  {fillMonth.map((el) => (
+                    <MenuItem key={el} value={el}>
+                      {el}
+                    </MenuItem>
+                  ))}
+                </Select>
+                <Select className={classes.dropDownDate} id="selectDate" value="0">
+                  <MenuItem value="0">Date</MenuItem>
+                  {fillDate.map((el) => (
+                    <MenuItem key={el} value={el}>
+                      {el}
+                    </MenuItem>
+                  ))}
+                </Select>
+                <Select className={classes.dropDownYear} labelId="label" id="select" value="10">
+                  <MenuItem value="10">Year</MenuItem>
+                  {fillYear.map((el) => (
+                    <MenuItem key={el} value={el}>
+                      {el}
+                    </MenuItem>
+                  ))}
+                </Select>
+              </Box>
+              <TextField
+                className={classes.testWidth}
+                id="username"
+                margin="normal"
+                InputLabelProps={{
+                  shrink: true,
+                }}
+                InputProps={{
+                  classes: { input: classes.inputs },
+                }}
+                name="email"
+                autoComplete="email"
+                autoFocus
+                placeholder="Your email"
+              />
+              <Box flexDirection="row" className={classes.phoneDetails}>
+                <span className={classes.phoneNumber}>No Phone number entered</span>
```

Comment:
Please use the MUI `Typography` component for this `span`

Reply:
Done the changes